content/index.de.md

```diff
@@ -0,0 +1,248 @@
+title: Einführung in das barrierefreie Web
+lang: de
+last_updated: 2023-11-26
+translators:
+- name: "Alexej Rotar"
+contributors:
+  - name: "Jens Oliver Meiert"
+    link: "https://meiert.com/"
+github:
+    repository: w3c/wai-intro-accessibility
+    branch: gh-pages
+    path: content/index.de.md
+
+permalink: /fundamentals/accessibility-intro/de
+ref: /fundamentals/accessibility-intro/
+changelog: /fundamentals/accessibility-intro/changelog/
+layout: default
+  
+# In the footer below:
+# Do not change the dates
+# Do not translate CHANGELOG
+# Translate the other words, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: >
+  <p><strong>Datum:</strong> Aktualisiert am 20. November 2023. Erstmals veröffentlicht 2005. CHANGELOG.</p>
+  <p><strong>Redakteur:</strong> <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Entwickelt von der Arbeitsgruppe für Bildung und Öffentlichkeitsarbeit. (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Zusammenfassung" class="full" %}
+{:/}
+
+Wenn Webseiten und Webwerkzeuge richtig gestaltet und programmiert sind, können Menschen mit Behinderungen sie nutzen. Derzeit werden jedoch viele Seiten und Werkzeuge mit Barrieren entwickelt, die ihre Nutzung für manche Menschen schwer oder unmöglich machen.
+
+Das Web barrierefrei zu machen dient Einzelnen, Unternehmen sowie der Gesellschaft. Internationale Webstandards legen fest, was für Barrierefreiheit erforderlich ist.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Inhaltsverzeichnis" class="full" %}
+{:/}
+
+-   Das Inhaltsverzeichnis wurde automatisch erstellt.
+{:toc}
+
+<span class="box-h box-h-simple box-h-full">Verwandte Quelle</span><br>
+{% include video-link.html title="Videoeinführung in das barrierefreie Web und W3C-Standards <em>(4 Minuten)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits/" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Barrierefreiheit im Kontext {#context}
+
+<blockquote class="pull">
+  <p>Die Stärke des Webs liegt in dessen Universalität.<br>
+    Der Zugriff aller Menschen, unabhängig von Behinderungen, ist ein unabdingbarer Bestandteil.</p>
+  <footer><cite>Tim Berners-Lee, Leiter des W3C und Erfinder des World Wide Web</cite></footer>
+</blockquote>
+
+Das Web ist grundlegend so gestaltet, dass es für alle Menschen unabhängig von deren Hardware, Software, Sprache, Ort oder Fähigkeiten funktioniert. Wenn das Web dieses Ziel erreicht, ist es für Menschen mit unterschiedlichem Hör-, Bewegungs-, Seh- und Denkvermögen zugänglich.
```

Comment:
@alexejrotar You sometimes use "Denkvermögen", sometimes "kognitive" to translate "cognitive".
Is there a specific reason for this? Do you know what word is more generally used in the German-speaking accessibility community?

FYI, I notice the use of kognitiven/kognitiver in the [Unofficial german translation of WCAG 2.1](https://outline-rocks.github.io/wcag/translations/WCAG21-de/)

Reply:
I suppose I could translate "cognitive ability" as "kognitive Fähigkeiten". I found "Denkvermögen" somewhat more natural, especially in conjunction with the rest of the sentence.